boards/nxp/mimxrt1170_evk/doc/index.rst

```diff
@@ -295,29 +296,29 @@ reprogrammed with JLink firmware.
 - MIMXRT1170-EVKB: :ref:`mcu-link-cmsis-onboard-debug-probe`
 - MIMXRT1170-EVK:  :ref:`opensda-daplink-onboard-debug-probe`
 
+Using LinkServer
+----------------
+
+LinkServer is the default runner for this board.
```

Comment:
I would just remove this so that it has zero chance to get stale, since default runner is automatically mentioned in the docs based on what board.cmake says. 

<img width="781" alt="image" src="https://github.com/user-attachments/assets/a753d9c9-58bf-4413-bbee-6d70d29aef1b" />


```suggestion
```